js/adapt-xapi.js

```diff
@@ -416,8 +416,9 @@ define(function(require) {
       if (!statement) {
         return;
       }
-
+      Adapt.trigger("xapi:willSendStatement", statement);
```

Comment:
Can you rename these events to `xapi:preSendStatement` and `xapi:postSendStatement` please?  Also, target at the `develop` branch.